README.md

```diff
@@ -66,6 +66,43 @@ for example, `tugraph/tugraph-env-centos7:1.1.0`, which can help developers get
 
 For more details, please refer to the docs in [doc-zh]
 
+## 4. Change Log
```

Comment:
I don't think the chang log of pr is necessary in this README, which only contains very high-level introduction.